app/src/main/java/com/framgia/forder/screen/shopmanagement/ItemShopInfoViewModel.java

```diff
@@ -11,10 +14,16 @@
 public class ItemShopInfoViewModel {
     private final Context mContext;
     private final ShopInfo mShopInfo;
+    private final ShopDomain mShopDomain;
+    public final ObservableInt mButtonJoinDomain;
```

Comment:
WARNING: Variable 'mButtonJoinDomain' must be private and have accessor methods.